<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceDialog</class>
 <widget class="QDialog" name="ReferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–°–ø—Ä–∞–≤–æ—á–Ω–∏–∫</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>–ü–æ–∏—Å–∫:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ —Ç–µ–∫—Å—Ç –¥–ª—è –ø–æ–∏—Å–∫–∞...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>üîç</string>
       </property>
       <property name="toolTip">
        <string>–ü—Ä–∏–º–µ–Ω–∏—Ç—å —Ñ–∏–ª—å—Ç—Ä –ø–æ–∏—Å–∫–∞</string>
       </property>
       <property name="maximumWidth">
        <number>40</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="sideButtonsLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>–î–æ–±–∞–≤–∏—Ç—å</string>
         </property>
         <property name="toolTip">
          <string>–î–æ–±–∞–≤–∏—Ç—å –Ω–æ–≤—É—é –∑–∞–ø–∏—Å—å (Insert)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>–ò–∑–º–µ–Ω–∏—Ç—å</string>
         </property>
         <property name="toolTip">
          <string>–ò–∑–º–µ–Ω–∏—Ç—å –≤—ã–±—Ä–∞–Ω–Ω—É—é –∑–∞–ø–∏—Å—å (Enter –∏–ª–∏ –¥–≤–æ–π–Ω–æ–π –∫–ª–∏–∫)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>–£–¥–∞–ª–∏—Ç—å</string>
         </property>
         <property name="toolTip">
          <string>–£–¥–∞–ª–∏—Ç—å –≤—ã–±—Ä–∞–Ω–Ω—É—é –∑–∞–ø–∏—Å—å (Delete)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reportButton">
         <property name="text">
          <string>–û—Ç—á—ë—Ç</string>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>–û–±–Ω–æ–≤–∏—Ç—å</string>
         </property>
         <property name="toolTip">
          <string>–û–±–Ω–æ–≤–∏—Ç—å –¥–∞–Ω–Ω—ã–µ –∏–∑ –±–∞–∑—ã (F5)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maximizeButton">
         <property name="text">
          <string>–†–∞–∑–≤–µ—Ä–Ω—É—Ç—å</string>
         </property>
         <property name="toolTip">
          <string>–†–∞–∑–≤–µ—Ä–Ω—É—Ç—å –æ–∫–Ω–æ –Ω–∞ –≤–µ—Å—å —ç–∫—Ä–∞–Ω (F11)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>–ó–∞–∫—Ä—ã—Ç—å</string>
         </property>
         <property name="toolTip">
          <string>–ó–∞–∫—Ä—ã—Ç—å –æ–∫–Ω–æ (Esc)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusLayout">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>–ó–∞–ø–∏—Å–µ–π: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
